services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> QuerySet:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(
+        session_date: datetime = None
+) -> QuerySet | MovieSession:
+    if session_date:
+        return MovieSession.objects.filter(
+            show_time__date=datetime.strptime(session_date, "%Y-%m-%d").date())
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> QuerySet:

```

Comment:
The return type hint for `get_movie_session_by_id` should be `MovieSession` instead of `QuerySet`, as `MovieSession.objects.get()` returns a single `MovieSession` instance.